Doc/library/sqlite3.rst

```diff
@@ -1345,27 +1347,23 @@ The following example illustrates the implicit and explicit approaches:
 .. literalinclude:: ../includes/sqlite3/converter_point.py
 
 
-Default adapters and converters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There are default adapters for the date and datetime types in the datetime
-module. They will be sent as ISO dates/ISO timestamps to SQLite.
+Default adapters and converters (deprecated)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The default converters are registered under the name "date" for
-:class:`datetime.date` and under the name "timestamp" for
-:class:`datetime.datetime`.
-
-This way, you can use date/timestamps from Python without any additional
-fiddling in most cases. The format of the adapters is also compatible with the
-experimental SQLite date/time functions.
+.. note::
 
-The following example demonstrates this.
+   The default adapters and converters are deprecated as of Python 3.12.
+   Use the :ref:`sqlite3-adapter-converter-recipes`
```

Comment:
```suggestion
   Use instead the :ref:`sqlite3-adapter-converter-recipes`
```

Reply:
Second one, as you presumably found out; the first one is not the right word order in English (though it is in a number of other languages). Just for reference, another valid alternative is

```python
Use the :ref:`sqlite3-adapter-converter-recipes` instead.
```

Thought I think I slightly prefer the wording you went with, since it sounds a bit gentler.